charts/pipelines-library/templates/pipelines/helm/gerrit-build-default.yaml

```diff
@@ -53,6 +53,21 @@ spec:
 
 {{- include "gerrit-build-start" . | nindent 4 }}
 
+    - name: check-chart-name
+      params:
+        - name: CODEBASE_NAME
+          value: $(params.CODEBASE_NAME)
+        - name: chart-dir
```

Comment:
do we need CHART_DIR?

Reply:
yes, we take a variable from the pipeline and pass it to the task, the same approach as for helm-docs, helm-lint, helm-template